app/src/main/java/com/example/notebook/presentation/ui/fragments/my_notes/MyNotesFragment.kt

```diff
@@ -40,8 +42,13 @@ SearchView.OnQueryTextListener {
 
     override fun onStart() {
         super.onStart()
-        viewModel.notes.observe(viewLifecycleOwner) {
-            notesAdapter.submitList(it)
+//        viewModel.allNotes.observe(viewLifecycleOwner) {
+//            notesAdapter.submitList(it)
+//        }
+        lifecycleScope.launch {
+            viewModel.allNotes.collect { notes ->
+                notesAdapter.submitList(notes)
+            }
```

Comment:
I would rather avoid collecting values this way, I am not sure what the difference is but we use this approach in our prod code: 
```
viewModel.yourFlow.flowWithLifecycle(viewLifecycleOwner.lifecycle)
            .onEach { data ->
                // do something with your data here
            }
            .launchIn(viewLifecycleOwner.lifecycleScope)
```